demo/kubernetes/rook-pool.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: rook.io/v1beta1
+apiVersion: rook.io/v1alpha1
```

Comment:
i had a separate sample to make it more obvious at a glance that a pool can be created with the tpr. It's not just for the storageclass sample.